src/components/block/InteractiveMap/MapKey.module.css

```
@@ -0,0 +1,22 @@
+.mapKey {
+  background-color: #fff;
```

Could you replace this with our color variable, [`--color-white`](https://github.com/ShelterTechSF/sheltertech.org/blob/d42e37c4f512b140ba30698285203022a267b124/src/stylesheets/global.css#L17), for the same color value? This helps us prevent the creation of ad hoc hardcode values, and perhaps would make it easier to globally change the color scheme in the future, for example, but choosing a slightly different shade for "white".

```suggestion
  background-color: var(--color-white);
```